Documentation/contributing/docs/docsstyle.rst

```diff
@@ -270,3 +311,167 @@ Roles
   .. code-block:: rst
 
     See `this GitHub issue <https://github.com/cilium/cilium/issues/1234>`__.
+
+Common pitfalls
+---------------
+
+There are best practices for writing documentation: try to follow them. In
+general, default to the `Kubernetes style guide`_, especially for `content best
+practices`_. Give particular attention to the recommendations from the
+following subsections.
+
+Use active voice
+~~~~~~~~~~~~~~~~
+
+Prefer::
+
+    Enable the flag.
+
+Avoid::
+
+    Ensure the flag is enabled.
+
+Use present tense
+~~~~~~~~~~~~~~~~~
+
+Prefer::
+
+    The service returns a response code.
+
+Avoid::
+
+    The service will return a response code.
+
+Address the user as "you", not "we"
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Prefer::
+
+    You can specify values to filter tags.
+
+Avoid::
+
+    We'll specify this value to filter tags.
+
+Use plain, direct language
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Prefer::
+
+    Always configure the bundle explicitly in production environments.
+
+Avoid::
+
+    It is recommended to always configure the bundle explicitly in production environments.
+
+Write for good localization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Assume that what you write will be localized with machine translation. Figures
+of speech often localize poorly, as do idioms like "above" and "below".
+
+Prefer::
+
+    The following example
+    To assist this process,
+
+Avoid::
+
+    The example below
+    To give this process a boost,
+
+Define abbreviations
+~~~~~~~~~~~~~~~~~~~~
+
+Define abbreviations when you first use them on a page.
+
+Prefer::
+
+    Certificate authority (CA)
+
+Avoid::
+
+    CA
+
+Don't use Latin abbreviations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Prefer::
+
+    - For example,
+    - In other words,
+    - by following the ...
+    - and others
+
+Avoid::
+
+    - e.g.
+    - i.e.
+    - via
+    - etc.
+
+Spell words fully
+~~~~~~~~~~~~~~~~~
+
+Prefer::
+
+    and
+
+Avoid::
+
+    &
+
+Use specific wording rather than vague wording
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Prefer::
+
+    For each core, the Ingester attempts to spawn a worker pool
+
+Avoid::
+
+    For each core, it attempts to spawn a worker pool
+
+Use specific instructions rather than vague instructions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Prefer::
+
+    Set the annotation value to remote
+
+Avoid::
+
+    Set it to remote
+
+.. _Kubernetes style guide: https://kubernetes.io/docs/contribute/style/style-guide/
+.. _content best practices: https://kubernetes.io/docs/contribute/style/style-guide/#content-best-practices
+
+Specific language
```

Comment:
What's the difference between this section and the sections "Use specific instructions rather than vague instructions" and "Use specific wording rather than vague wording"? I feel like those two sections are examples that could be put under this section.

Reply:
@learnitall That's a good idea.

Reply:
True, I'll move them under this section. Thanks